src/components/FullTags/FullTagPage.vue

```diff
@@ -1,23 +1,22 @@
 <template>
   <div class="canvas-size overflow">
-    <div v-if="imageUrls.length === 0" class="loader central-loader-position" />
-    <div
-      v-if="imageUrls.length === 0"
-      class="rotated-half-circle central-loader-position"
-    />
```

Comment:
Why have you removed this part. If I remember well, it's a loader in case of the loading is slow?

Reply:
It was a mistake, i put it back.